go/vt/binlog/keyspace_id_resolver.go

```diff
@@ -34,7 +34,7 @@ import (
 	topodatapb "vitess.io/vitess/go/vt/proto/topodata"
 )
 
-var useV3ReshardingMode = flag.Bool("binlog_use_v3_resharding_mode", true, "True iff the binlog streamer should use V3-style sharding, which doesn't require a preset sharding key column.")
+var useV3ReshardingMode = flag.Bool("binlog_use_v3_resharding_mode", true, "(DEPRECATED) True if the binlog streamer should use V3-style sharding, which doesn't require a preset sharding key column.")
```

Comment:
I think the `iff` was intentional - meaning `if-and-only-if`. But I'm fine with removing the whole description and just keeping `DEPRECATED`. Whatever change you make here needs to go into flags_test.go as well.